app/controllers/BulkRequestReceivedController.scala

```diff
@@ -39,24 +39,22 @@ class BulkRequestReceivedController @Inject()(authAction: AuthAction,
   def get = authAction.async {
       implicit request => {
         val link = request.linkId
-
         Logger.debug(s"[BulkRequestReceivedController][get][GET] : ${request.body}")
         sessionService.fetchGmpBulkSession().flatMap {
           case Some(session) if session.callBackData.isDefined =>
             val callbackData = session.callBackData.get
-
             bulkRequestCreationService.createBulkRequest(callbackData.collection, callbackData.id, session.emailAddress.getOrElse(""), session.reference.getOrElse("")) match {
 
-              case Left(bulkRequest) =>
-                gmpBulkConnector.sendBulkRequest(bulkRequest, link).map {
+              case Left(bulkRequest) => gmpBulkConnector.sendBulkRequest(bulkRequest, link).map {
                   case OK => Ok(views.html.bulk_request_received(bulkRequest.reference))
                   case CONFLICT => Ok(views.html.failure(Messages("gmp.bulk.failure.duplicate_upload"), Messages("gmp.bulk.problem.header"), Messages("gmp.bulk_failure_duplicate.title")))
                   case REQUEST_ENTITY_TOO_LARGE => Ok(views.html.failure(Messages("gmp.bulk.failure.too_large"), Messages("gmp.bulk.file_too_large.header"), Messages("gmp.bulk_failure_file_too_large.title")))
                   case _ => Ok(views.html.failure(Messages("gmp.bulk.failure.generic"), Messages("gmp.bulk.problem.header"), Messages("gmp.bulk_failure_generic.title")))
```

Comment:
Should this be the same as the Right(_) case? It's confusing that it renders a generic failure for Left but Right has a redirect